packages/hint/docs/user-guide/concepts/parsers.md

```diff
@@ -1,55 +1,66 @@
+written.
 
-You can specify what `parser`s you want to use via the `.hintrc`
-configuration file:
+To utilize a parser, first install its package. The package name should
+begin with `@hint/parser-`, `webhint-parser-`, or
+`@namespace/webhint-parser-`. Once you've installed the appropriate
+package, specify which parsers you want to use by adding them to the
+`parsers` array in your .hintrc configuration file.
+
+For example, if you've installed `@hint/parser-example1` and
+`webhint-parser-example2`, add the following:
 
 ```json
 {
-    "parsers": ["parser1", "parser2"]
+    "parsers": [
+        "@hint/parser-example1",
+        "webhint-parser-example2"
+    ]
 }
 ```
 
-## List of official `parser`s
+## List of official parsers
 
-* [`babel-config`][@hint/parser-babel-config] A `Babel configuration` parser
-  which validates the provided `json` so hints analyzing `.babelrc` files can
-  be built.
+* [`@hint/parser-babel-config`][babel-config] A Babel configuration
+  parser which validates the provided json so hints analyzing .babelrc
+  files can be built.
 
-* [`css`][@hint/parser-css]: A `CSS` parser built on top of
-  [PostCSS][postcss] so hints can analyze `CSS` files.
+* [`@hint/parser-css`][css]: A CSS parser built on top of [PostCSS][] so
+  hints can analyze CSS files.
 
-* [`html`][@hint/parser-html]: An `HTML` parser built on top of `jsdom`.
-  This parser is only necessary if you are using the `local connector`
-  and analyzing local `HTML` files. Otherwise the related `HTML` events
-  are taken care directly by the other `connector`s.
+* [`@hint/parser-html`][html]: An HTML parser built on top of jsdom.
+  This parser is only necessary if you are using the [local connector][]
+  and analyzing local HTML files. Otherwise the related HTML events are
+  taken care directly by the other connectors.
 
-* [`javascript`][@hint/parser-javascript]: A `JavaScript`
-  parser built on top of `ESLint` so hints for analyzing `JavaScript`
-  files can be built.
+* [`@hint/parser-javascript`][javascript]: A JavaScript parser built on
+  top of ESLint so hints for analyzing JavaScript files can be built.
 
-* [`manifest`][@hint/parser-manifest]: A parser that validates if a
-  `web app manifest` is valid and emit information related to it.
+* [`@hint/parser-manifest`][manifest]: A parser that checks validity of
+  a web app manifest and emits information related to said manifest.
 
-* [`typescript-config`][@hint/parser-typescript-config]: A parser
-  that validates if the `TypeScript configuration` is valid.
+* [`@hint/parser-typescript-config`][typescript-config]: A parser that
+  checks validity of a TypeScript configuration.
 
 ## How to use a parser
 
-To use a parse you need to subscribe to the event(s) that the parser dispatches.
-Please check the details page of each parser to have more information about the
-events emitted by them.
+To utilize a parser when writing your own hints, subscribe to the
```

Comment:
I'm gonna be 100% honest, I'm not entirely convinced i understand parsers or how they're relevant to someone whose only interaction with webhint is consuming it so i hope i got this section right!

Reply:
You got them right!
TBH, we are debating hiding the parser configuration to regular users and load them automatically when a hint requires them.